### PR TITLE
9.18.4 RN

### PR DESCRIPTION
**Pull request by Adam-Dupaski**

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
For KI: add links to all published fix versions

**Comment by Adam-Dupaski** [comment]
Merge dev in for previous KI fixes

File: content/en/docs/releasenotes/studio-pro/9/9.18.md [hunk 1]
```diff
@@ -10,6 +10,63 @@ weight: 82
...
+
+* We fixed an issue where a native mobile app stays stuck on the splash screen after a user logs out (Ticket 171288)
+* We fixed an issue when Oops dialog appeared when opening Domain Model Editor in certain cases. (Ticket 170735, 170571)
+* We fixed an issue where creating a deployment package on the cloud fails for an app with OTA (Over-the-air updates) enabled. (Ticket 169390)
+* We fixed the 'space bar' functionality in the grid control for 'check-box' cells: The user can now toggle on/off multiple checkboxes (Ticket 165926).
+* We fixed an issue with switching to "None" option in close pages of "Show page" microflow activity. (Ticket 174834)
+* We fixed an issue in offline navigation profiles where retrieving an object that has a generalization to a base entity type results into returning the base type instead of the specialized type that has been retrieved. (Ticket 174251)
+* We fixed an issue where passing an object from a pluggable widget data source as an argument to a nanoflow data source could cause an error if the nanoflow tried to change an attribute of the object. (Ticket 174605)
+
```
Comment (tamelbrun) [inline content/en/docs/releasenotes/studio-pro/9/9.18.md:50]: @Adam-Dupaski I have an additional release note that didn't make the freeze:

```suggestion
* We fixed an issue where an argument of an *On change* action that was mapped to a snippet parameter could give an error dialog when trying to deploy the app. (Ticket 174249)
```

Comment (Adam-Dupaski) [reply content/en/docs/releasenotes/studio-pro/9/9.18.md:50]: Added, thanks: https://github.com/mendix/docs/pull/5509/commits/6aeedaa96ba15c8337e50059554f896d1503577d

**Pull request merged**